TicketBooking.Data/Migrations/20230223093705_createRole.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace TicketBooking.Data.Migrations
+{
+    public partial class createRole : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
```

Comment:
file name is create role but the table name is refreshToken. It's nnot make sense.